### ISSUE TITLE
Outdated flow-runtime on npm

### ISSUE THREAD

**Issue by artem-barmin-dev** [codemix/flow-runtime#231]
Looks like npm version (0.17) is 2 versions behind releases on Github(0.19). Why they are not published?

**Comment by dod91** [comment]
I agree, it would be nice to see the versions tagged on Github on npm as well.

**Comment by jedwards1211** [comment]
The project's pretty much dead because of flow's declining popularity and because flow-runtime was overambitious in the first place (trying to check all type annotations everywhere at runtime was never a wise idea).  If you want to use `flow-runtime` to validate API inputs you could use my packages [`typed-validators`](https://github.com/jcoreio/typed-validators) and [`gen-typed-validators`](https://github.com/jcoreio/gen-typed-validators) instead.  These packages are a lot like the more popular [zod](https://github.com/colinhacks/zod) but support Flow in addition to TypeScript.
